### DISCUSSION TITLE
Tile Editing Mode

### DISCUSSION THREAD

**Discussion by funmise** [rxhanson/Rectangle#1535] [Q&A]
Is there a way to enter a Tile Editing Mode like ubuntu does for tiling? By that I mean a shortcut that allows me to switch focus between the tiled windows on a desktop/workspace? I found AltTab and in combination with rectangle everything is almost perfect, but I'd like to be able to just quickly switch focus based on what  desktop I am.
[Screencast from 2024-12-29 14-27-57.webm](https://github.com/user-attachments/assets/5722efd9-7293-40b8-aea6-b8155f90dc07)

**Comment by rxhanson** [comment]
At the moment, this is not in Rectangle, but I would welcome a pull request that adds this in.
